platform/darwin/src/MGLTypes.h

```diff
@@ -70,6 +70,21 @@ typedef NS_OPTIONS(NSUInteger, MGLMapDebugMaskOptions) {
 #endif
 };
 
+/**
+ A structure containing information about a transition.
+ */
+typedef struct MGLTransition {
+    /**
+     The duration in seconds to animate any changes to the style URL or to layout and paint attributes.
```

Comment:
It's a bit of a nonsequitur to talk about style URLs and attributes here. This structure is generic enough that its documentation should be more self-contained. For example:

> The amount of time the animation should take, not including the `delay`.

(It doesn't include the delay, does it?)